tools/cabana/dbc/dbc.cc

```diff
@@ -78,6 +78,9 @@ QString cabana::Msg::newSignalName() {
 }
 
 void cabana::Msg::update() {
+  if (transmitter.isEmpty()) {
+    transmitter = "XXX";
+  }
```

Comment:
This function doesn't seem to be called when you exit the message edit dialog so you see `()` until you edit a signal

Reply:
my bad. This is only called when parsing the dbc file.

Reply:
fixed